package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vue-fineuploader",
-  "version": "2.0.0-rc.6",
+  "version": "2.1.0-rc.6",
```

Comment:
This was supposed to be a release candidate until all the components are finished, and be officially pushed in 2.0.0

Thus, incremented the `rc`, not the minor version.